chore: Try to download Z3 5 times instead of just once (#1835)
dafny-lang/dafny: RELEASE_NOTES.md

# Upcoming

> - fix: Dafny with JavaScript target not producing any output on Windows (https://github.com/dafny-lang/dafny/pull/1824)

Reviewer: ```suggestion
- fix: No output when compiling to JavaScript on Windows (https://github.com/dafny-lang/dafny/pull/1824)

```
